src/pyff/schema/saml-metadata-trustinfo-v1.0.xsd

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema  
+ targetNamespace="https://seamlessaccess.org/NS/trustinfo"
+ xmlns="http://www.w3.org/2001/XMLSchema"
+ xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"
+ xmlns:ti="https://seamlessaccess.org/NS/trustinfo"
+ elementFormDefault="unqualified"
+ attributeFormDefault="unqualified"
+ blockDefault="substitution"
+ version="1.0">
+
+ <annotation>
+   <documentation>
+     Document title: Metadata Extension Schema for SAML V2.0 Metadata Extensions for Trust Information 1.0
+     Document identifier: metadata-trustinfo-v1.0.xsd
+     Location: http://docs.oasis-open.org/security/saml/Post2.0/
+     Revision history:
+   </documentation>
+ </annotation>
+
+ <import namespace="urn:oasis:names:tc:SAML:2.0:metadata"
+   schemaLocation="saml-schema-metadata-2.0.xsd"/>
+ <import namespace="http://www.w3.org/XML/1998/namespace"
+   schemaLocation="xml.xsd"/>
+
+ <element name="TrustInfo" type="ti:TrustInfoType"/>
+ <complexType name="TrustInfoType">
+   <sequence>
+     <element ref="ti:MetadataSource" minOccurs="0" maxOccurs="unbounded"/>
+     <element ref="ti:TrustProfile" minOccurs="0" maxOccurs="unbounded"/>
+     <element ref="ti:TrustProfileRef" minOccurs="0" maxOccurs="unbounded"/>
+   </sequence>
+ </complexType>
+
+ <element name="MetadataSource" type="anyURI"/>
+
+ <element name="TrustProfileRef" type="anyURI"/>
+
+ <element name="TrustProfile" type="ti:TrustProfileType"/>
+ <complexType name="TrustProfileType">
+    <sequence minOccurs="0">
```

Comment:
Why `minOccurs="0"`? Surely a trust profile must have some information about the entities that the SP trusts?

Reply:
You are right, changing it to 1